paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -544,6 +557,17 @@ internal fun PaymentOptionUi(
             )
         }
         if (isEditing && onRemoveListener != null) {
+            val openDialog = remember { mutableStateOf(false) }
+
+            SimpleDialogElementUI(
+                openDialog = openDialog,
+                titleText = onRemoveTitle,
```

Comment:
what about onRemoveTitle -> RemovePmDialogTitle?

Reply:
I like that better. I pushed the rename.